docs/openrpc.json

```
@@ -5,6 +5,28 @@
         "description": "A specification of the implemented Ethereum JSON RPC APIs interface for Hedera clients and adheres to the Ethereum execution APIs schema.",
         "version": "0.23.0-SNAPSHOT"
     },
+    "servers": [
+        {
+          "name": "Mainnet",
+          "url": "https://mainnet.hashio.io/api",
+          "description": "Hedera Mainnet endpoint, hosted by [Hashio](https://swirldslabs.com/hashio/), a JSON-RPC Relay Community Service. Global rate limits apply."
```

FOr this and the testnet and previewnet
```suggestion
          "description": "Hedera Mainnet endpoint, hosted by [Hashio](https://swirldslabs.com/hashio/), a JSON-RPC Relay Community Service. Global and HBAR spend rate limits apply."
```

---

done!